docs/en/02_Developer_Guides/03_Forms/Field_types/05_UploadField.md

```diff
@@ -60,52 +78,60 @@ Enable multiple fileuploads by using a many_many (or has_many) relation. Again,
 				$uploadField = new UploadField(
 					$name = 'GalleryImages',
 					$title = 'Upload one or more images (max 10 in total)'
-				)	
+				)
 			);
+
+			$uploadField->setAllowedFileCategories('image');
 			$uploadField->setAllowedMaxFileNumber(10);
 			
 			return $fields;			
 		}	
 	}
-	class GalleryPage_Controller extends Page_Controller {
-	}
-```
 
-```php
+
+	:::php
 	class GalleryImageExtension extends DataExtension {
 		private static $belongs_many_many = array('Galleries' => 'GalleryPage);
 	}
-```
 
-```yml
-  Image:
-    extensions:
-        - GalleryImageExtension
-```
+
+	:::yaml
+	Image:
```

Comment:
`Image` (the class) doesn't exist now right (or is deprecated), so this would have to be on `File`.